### PR TITLE
Add default profile picture for users who do not set one

### PR DESCRIPTION
**Pull request by SleeplessOne1917**
Addresses #1162. Unsure if I should open a separate issue for allowing instance admins to set their own default profile picture if they want.

Here is a post made by a user with a default pfp.
image

### CODE REVIEW

File: src/shared/components/person/person-details.tsx [hunk 1]
```diff
@@ -107,16 +107,6 @@ export class PersonDetails extends Component<PersonDetailsProps, any> {
     setupTippy();
   }
 
-  // TODO wut?
```
Comment (SleeplessOne1917) [inline src/shared/components/person/person-details.tsx:110]: This was 3 years old, commented out, and I have not idea what to make of "TODO wut?" so I removed it.

**Review by dessalines — APPROVED** [review]

**Comment by dessalines** [comment]
I haven't been a big fan of slamming in pfps where people don't set them, but if enough people want them to make things line up better, I'm not opposed.

**Comment by SleeplessOne1917** [comment]
@dessalines There was a discussion about this in the dev channel. Do you think [gravatars](https://en.gravatar.com/site/implement/images/) might be a better approach?

**Comment by krestenlaust** [comment]
Something like gravatars would be pretty cool to have

**Pull request merged**